TAF/testScenarios/functionalTest/V2-API/core-data/event/DELETE-positive.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Resource         TAF/testCaseModules/keywords/commonKeywords.robot
+Suite Setup      Setup Suite
+
+*** Variables ***
+${SUITE}         Core-Data Event DELETE Positive Testcases
+
+*** Test Cases ***
+EventDELETE001 - Delete event by ID
+    Given Create An Event
+    When Delete Event By ID
+    Then Should Return Status Code "204"
+    Then Event Should Be Deleted
+
+EventDELETE002 - Delete events with specified device by device id
+    Given Create Multiple Events
+    When Delete Events With Specified Device
+    Then Should Return Status Code "204"
+    Then Events With Specified Device Should Be Deleted
+
+EventDELETE003 - Delete all old events (and associated readings) based on delimiting age
+    Given Create Multiple Events
+    When Delete Events By Age
+    Then Should Return Status Code "204"
+    Then Old Events Should Be Deleted
+
+EventDELETE004 - Delete all events (and associated readings)
+    Given Create Multiple Events
+    When Delete Events
+    Then Should Return Status Code "204"
+    Then All Events Should Be Deleted
```

Comment:
- The API of /event/scrub is for removing all pushed events. If this test is for the API, the validation needs to be changed to "All Pushed Events" and add new validation to verify the associated readings are deleted.

- Needs to modify the second "Then" to "And" for all tests on this file.